cardboard/templates/home.html

```diff
@@ -24,12 +24,12 @@ <h1>Cardinality Cardboard</h1>
                 </li>
                 <li>
                     <b>How are you using my data?</b><br>
-                    We match your email against a whitelist of emails for each hunt. The actions you take while using the Cardboard (such as solving a puzzle) may be logged and attributed to your email. This data is only used for logging purposes. See <a href="{% url 'privacy'%}">here</a> for our full privacy policy.
+                    We match your email against an allowlist of emails for each hunt. The actions you take while using the Cardboard (such as solving a puzzle) may be logged and attributed to your email. This data is only used for logging purposes. See <a href="{% url 'privacy'%}">here</a> for our full privacy policy.
```

Comment:
Consider replacing with `an approved list of emails...` to be more specific.

Reply:
Done.